packages/plugin-ext/src/main/browser/view/tree-view-widget.tsx

```diff
@@ -372,4 +372,24 @@ export class TreeViewWidget extends TreeWidget {
             }
         }
     }
+
+    protected render(): React.ReactNode {
+        return React.createElement('div', this.createContainerAttributes(), this.renderTreeInfo(), this.renderTree(this.model));
+    }
+
+    private _message: string | undefined;
+    get message(): string | undefined {
+        return this._message;
+    }
+
+    set message(message: string | undefined) {
+        this._message = message;
+        this.update();
+    }
+
+    protected renderTreeInfo(): React.ReactNode {
+        if (this._message) {
+            return <div className='search-info'>{this._message}</div>;
```

Comment:
we should not rely on classes of completely other components

Reply:
Can anybody help with searching/selection of suitable class here?

Reply:
You should be able to define a new css class?